service_mesh/servicemesh-release-notes.adoc

```diff
@@ -17,4 +17,8 @@ include::modules/ossm-rn-new-features.adoc[leveloffset=+1]
 
 include::modules/ossm-rn-known-issues.adoc[leveloffset=+1]
 
+include::modules/jaeger-rn-known-issues.adoc[leveloffset=+2]
```

Comment:
Did you add a Jaeger item to the topic.yml? Should this go there instead of in the sm release notes?

Reply:
I'm single sourcing, publishing the same info in two sets of release notes.